repositories/fedora-30.json

```diff
@@ -0,0 +1,18 @@
+{
+    "x86_64": [
+        {
+            "id": "fedora",
+            "name": "Fedora 30",
```

Comment:
Unrelated to this patch, but why are we including the `name` everywhere? It's not really used, is it?

Reply:
Yeah, I was going to ask you the same question ;) Should look into that (separately).